app/helpers/records_helper.rb

```diff
@@ -70,6 +71,34 @@ def get_result_values(records, measures, pop_set, pop_keys, key)
     end.to_h
   end
 
+  # This method returns a hash of the form
+  #  {BSON::ObjectId('1')=>[[75]],
+  #  BSON::ObjectId('2')=>[[]],
+  #  BSON::ObjectId('3')=>[[50][10]]}
+  #
+  # Inputs:
+  # records: An array of records (If you only care about a single record then pass in [record])
+  # measures: An array of measures (If you only care about a single measure then pass in [measure])
+  # pop_set: The population set identifier for the result set you care about, for example ['PopulationCriteria1']
+  # key: The name of the element which should be the key of the key value pair returned.
+  # We currently use 'patient_id' and 'measure_id' as keys and then do lookups for the values
+  # in the hash when generating miscellaneous views around the Master Patient List.
+  #
+  # Usage tip: It is strongly recommended you pass in either an array of records OR and array
```

Comment:
I think I get where the comment is going with this.

the returned hash 'key' will be the patient id, or the measure id.  Ideally, it will be the key of the thing you pass in the array of (i.e., patient or measure).  If you pass in an array of both, the key/value pair is less useful.